target/linux/ath79/dts/ar7100_mikrotik_routerboard-4xx.dtsi

```diff
@@ -0,0 +1,139 @@
+// SPDX-License-Identifier: GPL-1.0-or-later OR MIT
+
+/dts-v1/;
```

Comment:
This only belongs into the DTS. Remove it here.

Reply:
Deleted